_develop/connect/get-started.md

```diff
@@ -162,4 +157,24 @@ if (userSession.isSignInPending()) {
   const userData = await userSession.handlePendingSignIn();
   // your user is now logged in.
 }
-```
\ No newline at end of file
+```
+
+## Design Guidance
```

Comment:
should this maybe be a separate page? seems quite important but is getting lost in the lengthy guide content

Reply:
Good idea, I'll break it out into its own page.